library/src/main/java/com/getstream/sdk/chat/view/messageinput/AttachmentsController.kt

```diff
@@ -65,18 +65,17 @@ internal class AttachmentsController(
                     isMedia
                 )
             }
-            selectedAttachments = filterMediaFiles(isMedia, selectedAttachments)
             setSelectedAttachmentAdapter(messageInputType, isMedia)
             view.showLoadingTotalAttachments(false)
         }
     }
 
-    private fun filterMediaFiles(isMedia: Boolean, filesToFilter: Set<AttachmentMetaData>): Set<AttachmentMetaData> {
-        return if (!isMedia) {
-            filesToFilter
-        } else {
-            filesToFilter.filter(mediaAttachmentsPredicate).toSet()
-        }
+    private fun filterMediaFiles(
```

Comment:
Naming of this function doesn't relate its' logic anymore. Before it filtered only media files, now for media and non-media